modules/nf-core/kallisto/quant/main.nf

```diff
@@ -27,12 +29,16 @@ process KALLISTO_QUANT {
     prefix = task.ext.prefix ?: "${meta.id}"
     def gtf_input = gtf ? "--gtf ${gtf}" : ''
     def chromosomes_input = chromosomes ? "--chromosomes ${chromosomes}" : ''
+    def single_end_params = meta.single_end ? "--single --fragment-length=$fragment_length --sd=$fragment_length_sd" : ''
```

Comment:
How does this behave if fragment_length and/or fragment_length_sd are empty for single ended data? Do we need to catch this case in the module code?

Reply:
Good point- checks added